app/graphql/api_schema.rb

```diff
@@ -8,4 +8,13 @@ class ApiSchema < GraphQL::Schema
 
   # Add built-in connections for pagination
   use GraphQL::Pagination::Connections
+
+  # Error handling
+  use GraphQL::Execution::Errors
+
+  # Validation errors
+  rescue_from(ActiveRecord::RecordInvalid) do |err, _obj, _args, _ctx, _field|
+    # Raise a graphql-friendly error with a custom message
+    raise GraphQL::ExecutionError, err.to_s
+  end
```

Comment:
This allows graphQL to display validation errors in responses. 

[source](https://graphql-ruby.org/errors/error_handling)

Reply:
Nice work on this.

Reply:
Nice work here!